PendingReleaseNotes

```diff
@@ -5,6 +5,42 @@
   python3.6 as the native python3. see the `announcement <https://lists.fedoraproject.org/archives/list/epel-announce@lists.fedoraproject.org/message/EGUMKAIMPK2UD5VSHXM53BH2MBDGDWMO/>_`
   for more details on the background of this change.
 
+* Because the content of telemetry reports has changed, you will need to
```

Comment:
Could we please drop this first bullet point, so we only have *one* telemetry release note?